system/jlib/jlog.cpp

```diff
@@ -917,13 +917,22 @@ void ComponentLogMsgFilter::addToPTree(IPropertyTree * tree) const
     tree->addPropTree("filter", filterTree);
 }
 
+
 bool RegexLogMsgFilter::includeMessage(const LogMsg & msg) const 
 { 
     if(localFlag && msg.queryRemoteFlag()) return false; 
     SpinBlock b(lock);
     return const_cast<RegExpr &>(regex).find(msg.queryText()) != NULL;
 }
 
+void WUIDLogMsgFilter::addToPTree(IPropertyTree * tree) const
```

Comment:
wuid should just a value of jobid, so why is a new filter type required? i.e. why isn't getJobLogMsgFilter suitable?

Reply:
This wasn't supposed to be checked-in, removing